ci-operator/config/openshift-priv/hypershift/openshift-priv-hypershift-main.yaml

```diff
@@ -73,32 +73,32 @@ releases:
       product: ocp
       stream: ci
       version: "4.19"
```

Comment:
Does this not go to 4.20 now?

Reply:
Did a quick huddle with Patryk. This was bumped previously to 4.19 when it should have been 4.18.